Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/FindPeakAutomaticTest.py

```diff
@@ -485,16 +485,6 @@ def test_process_returns_the_return_value_of_find_good_peaks(self):
             self.assertTableEqual(expected_return[0][1], actual_return[0][1])
             np.testing.assert_almost_equal(expected_return[1], actual_return[1])
 
```

Comment:
The tests don't fail on Windows and Ubuntu if matplotlib.pyplot is imported anymore. They are not nessesary anymore since @Pasarus said they might be from a time where we had different versions of mpl on different Operating Systems.

Reply:
So with matplotlib 3.8 `pyplot` is imported, but it isn't with 3.7, which causes this test to fail? If the other tests pass then deleting the test seems fine.

Reply:
Yes, I believe that was the conclusion. All other tests pass.